api/illumio/cloud/k8scluster/v1/resource_config.proto

```diff
@@ -94,8 +94,56 @@ message KubernetesResourceMutation {
   }
 }
 
+// Enumeration to define the different levels of logging.
+enum LogLevel {
+  // Default unspecified log level.
+  LOG_LEVEL_UNSPECIFIED = 0;
+  // Debug log level, useful for development and troubleshooting.
+  LOG_LEVEL_DEBUG = 1;
+  // Informational log level for general application information.
+  LOG_LEVEL_INFO = 2;
+  // Warning log level for potentially problematic situations.
+  LOG_LEVEL_WARN = 3;
+  // Error log level indicating something went wrong.
+  LOG_LEVEL_ERROR = 4;
+}
+
+// Message representing a log entry.
+message LogEntry {
+  // The log level of the log entry.
+  LogLevel level = 1;
+  // The timestamp when the log entry was recorded.
+  google.protobuf.Timestamp time = 2;
```

Comment:
You don't need those. The server now extracts the level from the `json_message`, and it doesn't use the `time`.
```suggestion
```